locals.tf

```diff
@@ -54,5 +54,11 @@ locals {
       time_window = "P2D"
       frequency   = "PT5M"
     }
+    "alr-prd-Monitoring-winux-law-custlog-warn-01" : {
```

Comment:
Is that the custlog part of the naming convention? I would put CustomLog as the logical name and `logsea` as the type, since it technically searches for logs in a table.

Reply:
the naming convention states custlog at the type where logsea also is an option, but since it was a CustomLog I went with that. 
But I do agree with your suggestion, so i will edit this.